be/src/exprs/agg/java_udaf_function.cpp

```diff
@@ -74,8 +74,10 @@ Status init_udaf_context(int64_t id, const std::string& url, const std::string&
     auto& state_clazz = JVMFunctionHelper::getInstance().function_state_clazz();
     ASSIGN_OR_RETURN(auto instance, state_clazz.newInstance());
     ASSIGN_OR_RETURN(auto get_func, analyzer->get_method_object(state_clazz.clazz(), "get"));
+    ASSIGN_OR_RETURN(auto batch_get_func, analyzer->get_method_object(state_clazz.clazz(), "batch_get"));
     ASSIGN_OR_RETURN(auto add_func, analyzer->get_method_object(state_clazz.clazz(), "add"));
-    udaf_ctx->states = std::make_unique<UDAFStateList>(std::move(instance), std::move(get_func), std::move(add_func));
+    udaf_ctx->states = std::make_unique<UDAFStateList>(std::move(instance), std::move(get_func),
```

Comment:
should we take compatibility into consideration? is there problem when rolling-updating SR

Reply:
lib/starrocks_be and udf-extends.jar will be the same version. so we don't have to consider about rolling upgrade problem